cmd/kubeadm/app/cmd/cmd.go

```diff
@@ -77,6 +77,7 @@ func NewKubeadmCommand(in io.Reader, out, err io.Writer) *cobra.Command {
 			}
 			return nil
 		},
+		Args: cobra.NoArgs,
```

Comment:
actually i think we shouldn't add this to the parent kubeadm command as it propagates to all sub-commands and we do have args in places.

Reply:
Ah yeah I missed the `AddCommand`s below.  I was going with a soft rule of "no `AddCommand`s and no `args` processing."